server/models/girl.js

```diff
@@ -0,0 +1,46 @@
+const Sequelize = require('sequelize')
+const sequelize = require('./connection');
+
+module.exports =  sequelize.define('girl', {
+    id: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    user_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    first_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    last_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    school_id: {
+      type: Sequelize.INTEGER,
+      allowNull: true,
+      references: {
+        model: 'school',
+        key: 'id'
+      }
+    },
+    birthdate: {
+      type: Sequelize.DATEONLY,
+      allowNull: true
+    },
+    password: {
+      type: Sequelize.TEXT,
+      allowNull: true
```

Comment:
I think password should be required in the schema (`allowNull: false`).

Reply:
done